EIPS/eip-7933.md

```diff
@@ -0,0 +1,152 @@
+---
+eip: 7933
+title: Ethereum Intent URI (EIURI)
+description: A Universal action URI format for Ethereum (represent and trigger Ethereum RPC requests)
+author: Frani (@frani)
+discussions-to: https://ethereum-magicians.org/t/eip-7924-ethereum-intent-uri-ei-uri-a-universal-action-uri-format-for-ethereum/23554
```

Comment:
eip number and eip number in link doesn't match,

Reply:
thanks. now I am going to update it